internal/infrastructure/rdb/place.go

```diff
@@ -653,6 +671,7 @@ func (p PlaceRepository) findByGooglePlaceId(ctx context.Context, exec boil.Cont
 		qm.Load(generated.GooglePlaceRels.GooglePlacePhotoAttributions),
 		qm.Load(generated.GooglePlaceRels.GooglePlaceReviews),
 		qm.Load(generated.GooglePlaceRels.GooglePlaceOpeningPeriods),
+		qm.Load(generated.GooglePlaceRels.Place+"."+generated.PlaceRels.PlacePhotos),
```

Comment:
[imo] このような順番になっているとまとまっていて見やすそうです。
```suggestion
                qm.Load(generated.GooglePlaceRels.Place),
                qm.Load(generated.GooglePlaceRels.Place+"."+generated.PlaceRels.PlacePhotos),
		qm.Load(generated.GooglePlaceRels.GooglePlaceTypes),
		qm.Load(generated.GooglePlaceRels.GooglePlacePhotoReferences),
		qm.Load(generated.GooglePlaceRels.GooglePlacePhotos),
		qm.Load(generated.GooglePlaceRels.GooglePlacePhotoAttributions),
		qm.Load(generated.GooglePlaceRels.GooglePlaceReviews),
		qm.Load(generated.GooglePlaceRels.GooglePlaceOpeningPeriods),
```